pkgs/tools/admin/salt/default.nix

```diff
@@ -29,6 +29,7 @@ python3.pkgs.buildPythonApplication rec {
 
   patches = [
     ./fix-libcrypto-loading.patch
+    ./fix-newer-zeromq.patch
```

Comment:
Oh sorry, I wrote an inline comment and didn’t notice that it wasn’t submitted.
Can you use `fetchpatch` instead of copying the patch into nixpkgs?

Something like this should work:
```nix
(fetchpatch {
  url = "https://github.com/saltstack/salt/commit/6ec8b90e402ff3fa8f27c7da70ece898592416bc.patch";
  hash = "…";
})
```

Reply:
I *think* I did it, but I have no idea what I'm doing, nor how to test it.